spec/std/indexable_spec.cr

```diff
@@ -143,6 +143,50 @@ describe Indexable do
     end
   end
 
+  describe "#find" do
```

Comment:
Might be worth having some specs testing the negative offset context.

Reply:
added additional tests!